build/ci/pr/Jenkinsfile

```diff
@@ -72,24 +72,21 @@ pipeline {
                         }
                     }
                     steps {
-                        createConfig()
-                        sh """
-                            cat >deployer-config.yml <<EOF
-id: gke-ci
-overrides:
-  kubernetesVersion: "1.12"
-  clusterName: eck-pr-$BUILD_NUMBER
-  vaultInfo:
-    address: $VAULT_ADDR
-    roleId: $VAULT_ROLE_ID
-    secretId: $VAULT_SECRET_ID
-  gke:
-    gCloudProject: $GCLOUD_PROJECT
-EOF
-                            echo "Setting CRD_FLAVOR to trivial-versions to support K8s < 1.14"
-                            echo "CRD_FLAVOR = trivial-versions" >> .env
-                            make -C build/ci TARGET=ci-e2e ci
-                        """
+                        script {
+                            createConfig()
+                            createDeployerConfig()
+                            sh """
+                                echo "Setting CRD_FLAVOR to trivial-versions to support K8s < 1.14"
+                                echo "CRD_FLAVOR = trivial-versions" >> .env
+                            """
```

Comment:
Since #1895, our default k8s version is 1.13. So you can remove the two echo to use the default non-trivial version of the CRDs.

Reply:
Done